frontend/src/components/special/CopyIndicator.tsx

```diff
@@ -11,10 +11,11 @@ export const CopyIndicatorContainer = styled.div.attrs((props: CopyIndicator) =>
     transform: (!props.copied) ? 'translateY(-60px)' : null,
   },
 }))<CopyIndicator>`
-  position: absolute;
+  position: fixed;
   top: 20px;
   left: 50%;
   transition: transform 0.25s;
+  z-index: 5;
```

Comment:
Yeah that's a good question - I'm not too sure what a formal system might look like - maybe something in Theme with like different z-index values for default, front, extra front etc. could work? I agree that for now though this might be the simplest solution

Reply:
Sounds good, yeah we may add that in the future but for now this is OK.